src/css/generic/_remove_arrow_input.scss

```diff
@@ -0,0 +1,3 @@
+input::-webkit-calendar-picker-indicator {
```

Comment:
This should be in base css. Generic folder has no specific use and is only adding more confusion.